### ISSUE TITLE
How to handle an unlabeled class rather than "BG"? 

### ISSUE THREAD

**Issue by inkyusa** [matterport/Mask_RCNN#1004]
Hello,
I am training a model with training dataset (high precision but relatively low recall). As an example, assuming that 10 cars appear in an image but for some reasons (e.g., missing small cars), we only successfully annotated 6 cars as "car". The rest of 4 cars were labeled as "ignore" or similar, that I want to ignore in training phase.
I think that those 4 cars should not be "BG" because a classifier trained with this dataset will confuse between "car" and "BG" which in turn may lead to decrease the performance.

A similar example can be found from Segnet "https://github.com/alexgkendall/caffe-segnet/blob/344c113bf1832886f1cbe9f33ffe28a3beeaf412/src/caffe/layers/softmax_loss_layer.cpp#L72" that ignores a user-specified label in loss calculation in softmax.

Is there any similar functionality in Mask_RCNN or anyone confronted a similar issue?
Thanks.

**Comment by YalongLiu** [comment]
Detect all cars in the image and use opencv or other tools to filter out small targets may be a better way.

**Comment by inkyusa** [comment]
Thanks for the comment.
